keyboards/printedpad/oled.c

```diff
@@ -0,0 +1,1447 @@
+        0x20, 0x40, 0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0xcf, 0xa8, 0xf0, 0xf0, 0xf0, 0x10, 0xd1, 0xbe, 0xee, 0xee, 0xe5, 0x04, 0x04, 0x05, 0x05, 
+        0x04, 0x02, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+    };
+
+    // Array of all bitmaps for convenience. (Total bytes used to store images in PROGMEM = 13728)
+    const char* epd_bitmap_allArray[26] = {
+        epd_bitmap_monkeylong1,
+        epd_bitmap_monkeylong2,
+        epd_bitmap_monkeylong3,
+        epd_bitmap_monkeylong4,
+        epd_bitmap_monkeylong5,
+        epd_bitmap_monkeylong6,
+        epd_bitmap_monkeylong7,
+        epd_bitmap_monkeylong8,
+        epd_bitmap_monkeylong9,
+        epd_bitmap_monkeylong10,
+        epd_bitmap_monkeylong11,
+        epd_bitmap_monkeylong12,
+        epd_bitmap_monkeylong13,
+        epd_bitmap_monkeylong14,
+        epd_bitmap_monkeylong15,
+        epd_bitmap_monkeylong16,
+        epd_bitmap_monkeylong17,
+        epd_bitmap_monkeylong18,
+        epd_bitmap_monkeylong19,
+        epd_bitmap_monkeylong20,
+        epd_bitmap_monkeylong21,
+        epd_bitmap_monkeylong22,
+        epd_bitmap_monkeylong23,
+        epd_bitmap_monkeylong24,
+        epd_bitmap_monkeylong25,
+        epd_bitmap_monkeylong26
+    };
+
+    uint16_t frame_sizes[26] = {
```

Comment:
These are all the same size.  You could just declare the array size above, and use that same value.  Would probably reduce the size and memory usage by doing so.